### PR TITLE
chore: package readmes

### PR DESCRIPTION
**Pull request by domagojk**
Addresses stoplightio/platform-internal#6806

**Checklist**

- [ ] Tests added / updated
- [x] Docs added / updated

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by domagojk** [comment]
c/p from slack:

I've moved docs from guides to cli, rulestes, functions and formats packages and adjusted some links for it. But guide is broken for now since I don't know how to solve that, and I also wrote very basic list of packages in Contributing.md (some of them I don't know how to describe)
Overall I'm not too happy how it looks :slightly_smiling_face: I think I liked root level guide more than this 🤷

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,7 +33,7 @@ Yarn is a package manager for your code, similar to npm. While you can use npm t
 3. In your terminal, navigate to the directory you cloned Spectral into (check that you are on the `develop` branch).
 4. Install the dependencies: `yarn`
 5. Build Spectral: `yarn build`
-6. Run Spectral from your local installation: `yarn cli lint [openapi_spec_file]`
+6. Run Spectral from your local installation: `yarn cli cli lint [openapi_spec_file]`
```
Comment (philsturgeon) [inline CONTRIBUTING.md:36]: Is this correct?

Comment (domagojk) [reply CONTRIBUTING.md:36]: yea, first `cli` is a name of yarn workspace and a second one is the name of the command. But it sure does look odd. Maybe we should add `cd packages/cli && yarn cli lint [openapi_spec_file]` or create a custom `cli` command in root? @P0lip wdyt?

Comment (P0lip) [reply CONTRIBUTING.md:36]: How about https://github.com/stoplightio/spectral/commit/6a7a4e571d02dc2e66c66565c76576283b264ffd

File: packages/rulesets/docs/sharing-rulesets.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ extends:
   - https://raw.githubusercontent.com/openapi-contrib/style-guides/master/apisyouwonthate.yml
 ```
 
-As with any ruleset, you can pass these directly to the [Spectral CLI](./2-cli.md):
+As with any ruleset, you can pass these directly to the [Spectral CLI](../../cli/README.md):
```
Comment (P0lip) [inline packages/rulesets/docs/sharing-rulesets.md:41]: will this work when the package is published to NPM?
I doubt it'll - perhaps we link to meta.stoplight.io?

**Comment by P0lip** [comment]
@mnaumanali94 may we find some volunteers to get this over the finish line? 😆

**Pull request closed without merging**